src/utils/order.js

```diff
@@ -135,9 +135,13 @@ export const newOrder = (order) => ({
   paymentMethod: order.paymentMethod,
   userComment: order.userComment,
   isPaid: order.isPaid,
-  promoCodeId: order.promoCodeId
+  promoCodeId: order.promoCodeId,
+  paymentStatus: paymentStatusTransfer(order.isPaid, order.paymentStatus)
 });
 
+export const paymentStatusTransfer = (isPaid, paymentStatus = null) =>
+  isPaid ? 'PAID' : paymentStatus || 'CREATED';
```

Comment:
why do you need || 'CREATED', what paymentStatus can contain, it can`t be 'CREATED'?

Reply:
Status 'CREATED'  appear when we created an order first time and this order didn't pay by cash.
I understand.  Perhaps the best solution would be to change the status to "not paid"

Reply:
Status 'CREATED' appear when we created an order first time and this order didn't pay by cash.
I understand. Perhaps the best solution would be to change the status to "NOT PAID"

Reply:
As i understand, when you create an order his paymentStatus don`t change to create, I think his status should change to CREATED, and you will don`t need to use this || CREATED